<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoscopicDialog</class>
 <widget class="QDialog" name="StereoscopicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="titlebar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="titlebar_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="WorkLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>830</width>
           <height>830</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(4, 4, 4);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>0</y>
           <width>251</width>
           <height>281</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="title">
          <string>参数</string>
         </property>
         <widget class="QLineEdit" name="lineEditbatch">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>180</y>
            <width>71</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.8);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_50">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>76</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(0,0,0,0.5);</string>
          </property>
          <property name="text">
           <string>采样频率</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditrefined">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>80</y>
            <width>70</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.8);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_42">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>76</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(0,0,0,0.5);</string>
          </property>
          <property name="text">
           <string>曝光时间</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_51">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>180</y>
            <width>101</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(0,0,0,0.5);</string>
          </property>
          <property name="text">
           <string>批处理点数</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditexposure">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>70</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.8);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_46">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>76</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(0,0,0,0.5);</string>
          </property>
          <property name="text">
           <string>细化点数</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditfrequency">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>130</y>
            <width>70</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.8);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ButtonSave">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>80</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #006db2;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>230</y>
            <width>80</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: white;
border: 1px solid rgba(0,0,0,0.3);
border-radius: 4px;
color: rgba(0,0,0,0.5);</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Buttonbatchtest">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>180</y>
            <width>60</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #006db2;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Buttonexposuretest">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>30</y>
            <width>60</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #006db2;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Buttonrefinedtest">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>80</y>
            <width>60</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #006db2;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Buttonfrequencytest">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>130</y>
            <width>60</width>
            <height>34</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #006db2;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>280</y>
           <width>251</width>
           <height>281</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="title">
          <string>数据</string>
         </property>
         <widget class="QPushButton" name="pBtnHSave">
          <property name="geometry">
           <rect>
            <x>59</x>
            <y>160</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #008CBA;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>保存高度数据</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBtnGSave">
          <property name="geometry">
           <rect>
            <x>59</x>
            <y>100</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #008CBA;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>保存灰度数据</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBtnESave">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>220</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #008CBA;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>保存编码器数据</string>
          </property>
         </widget>
         <widget class="QPushButton" name="PBtn3DShow">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>121</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #008CBA;
border-radius: 4px;
color: white;</string>
          </property>
          <property name="text">
           <string>3D显示</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../CHOWTAIFOOK_Detection.qrc"/>
 </resources>
 <connections/>
</ui>
